### DISCUSSION TITLE
能否自定义配置API请求地址

### DISCUSSION THREAD

**Discussion by jiubashi** [immersive-translate/immersive-translate#301] [Ideas]
如题，这样可以解决大陆地区在未使用VPN的情况下请求openai API失败的情况

**Comment by theowenyoung** [comment]
现在就可以呀：

<img width="1263" alt="Screenshot 2023-03-31 at 19 01 53" src="https://user-images.githubusercontent.com/62473795/229103350-87c1b5f0-2b3b-423c-bde6-2cbf71c1e91a.png">
